r/rstudio/Dockerfiles/v1.3.1073/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM     ppc64le/adoptopenjdk:8-openj9-bionic
+MAINTAINER      "Priya Seth <sethp@us.ibm.com>", "Eshant Gupta <guptaeshant@gmail.com>"
+
+ENV     PATH /usr/lib/rstudio-server/bin/:$PATH
+ENV     DEBIAN_FRONTEND noninteractive
+ENV 	R_VERSION 4.0.2
+ENV 	RStudio_VERSION 1.3.1073
+RUN     rm -rf /var/lib/apt/lists/ && \
+        apt-get -y update && \
+        apt-get install -y ant apparmor-utils autotools-dev build-essential \
+            ca-certificates cmake fakeroot file g++ git haskell-platform \
+            libapparmor1 libbz2-dev libcurl4-openssl-dev libedit2 libicu-dev \
+            libpam-dev libpango1.0-dev libssl-dev libxslt1-dev \
+            libboost-all-dev pkg-config psmisc python-dev python-setuptools \
+            pandoc pandoc-citeproc unzip uuid-dev wget gfortran libreadline-dev \
+	    xorg-dev libbz2-ocaml-dev liblzma-dev \
+            libghc-pcre-light-dev zlib1g-dev sudo lsb-release
+RUN     wget https://cloud.r-project.org/src/base/R-4/R-${R_VERSION}.tar.gz && \
+        tar -xvzf R-${R_VERSION}.tar.gz && cd R-${R_VERSION} && \
+        ./configure LIBnn=lib  --enable-R-shlib --with-pcre1 && \
+        make -j4 && make -j4 install
+RUN     wget https://github.com/rstudio/rstudio/archive/v${RStudio_VERSION}.tar.gz && \
+        tar zxvf v${RStudio_VERSION}.tar.gz && \
+        mkdir -p /rstudio-${RStudio_VERSION}/build
+RUN     echo "exit 0" > /rstudio-${RStudio_VERSION}/dependencies/common/install-crashpad && \
+        cd /rstudio-${RStudio_VERSION}/dependencies/linux && \
+        sed -i s/amd64/ppc64el/g ./install-dependencies-bionic && \
```

Comment:
we may want to add a separate patch for adding Power (or better still generic multi-arch support here) - the patch can reside in this repo for now but we should also raise a PR with rstudio community and reference it here.

Reply:
I can raise it. But it was commented -
`# when installing dependencies to setup a dev environment, need to run install-crashpad
# via sudo; however, this causes problems if used when building a Dockerfile`

Reply:
ok, we can address the crashpad issue later, but should atleast fix this one "sed -i s/amd64/ppc64el/g ./install-dependencies-bionic && \"